src/Rings/orderings.jl

```diff
@@ -1394,6 +1394,16 @@ function is_mixed(ord::MonomialOrdering)
   return false
 end
 
+@doc Markdown.doc"""
+    is_total(ord::MonomialOrdering)
+
+Return `true` if `ord` is total ordering, `false` otherwise.
+"""
+function is_total(ord::MonomialOrdering)
+  m = canonical_matrix(ord)
+  return nrows(m) >= ncols(m)
```

Comment:
ok, because _canonical_matrix does row reductions. Maybe add a comment on this for better readbility in the future.

Reply:
I don't think the `canonical_matrix` is row-reduced.